modules/gitops-argo-cd-properties.adoc

```diff
@@ -11,19 +11,24 @@ The Argo CD Custom Resource consists of the following properties:
 [options="header"]
 |===
 |Name |Description |Default |Properties
+
 |`applicationInstanceLabelKey` |The `metadata.label` key name where Argo CD injects the app name as a tracking label.|`app.kubernetes.io/instance` |
 
-|`applicationSet`
-|`applicationSet` controller configuration options.
-| `_<object>_`
+|`aggregatedClusterRoles`    |The aggregation of several `ClusterRoles` for a cluster-scoped instance. | `__<empty>__` |
+
+|`applicationSet` |`applicationSet` controller configuration options. | `_<object>_`
 a|* `<image>` - The container image for the `applicationSet` controller. This overrides the `ARGOCD_APPLICATIONSET_IMAGE` environment variable.
   * `<version>` - The tag to use with the `applicationSet` container image.
+  * `<enabled>` - The flag to use to enable the Application Set controller during Argo CD installation.
+  * `<env>` - Specify the environment for Application Set controller pods.
   * `<resources>` - The container compute resources.
   * `<logLevel>` - The log level used by the Argo CD Application Controller component. Valid options are `debug`, `info`, `error`, and `warn`.
   * `<logFormat>` - The log format used by the Argo CD Application Controller component. Valid options are `text` or `json`.
   * `<parallelismLimit>` - The kubectl parallelism limit to set for the controller (`--kubectl-parallelism-limit` flag).
   * `sourceNamespaces` - The list of non-control plane namespaces for creating and managing Argo CD `ApplicationSet` resources in target namespaces.
   * `scmProviders` - The list of URLs of the allowed Source Code Manager (SCM) Providers.
+  * `<scmRootCAConfigMap>` - The name of the ConfigMap that stores the Gitlab SCM Provider's TLS certificate which will be mounted on the Application Set Controller at the "/app/tls/scm/cert" path.
+  * `webhookServer` - Defines the available options for the Application Set Webhook Server component.  
```

Comment:
Why do some fields start with `<` and `>` and some don't? What is the criteria used to determine which to use?

It seems like `<` and `>` are used in the case where the field is an object, rather than a string/int/bool, but if this is true there are cases where `<` and `>` are incorrectly applied (`env`, for example)

Reply:
Hey @jgwest - I am unsure about the reasoning behind putting this in brackets. Your logic seems a good reason why it may be put as is but for the time being, I would like to put all the properties under brackets. When we attempt to restructure and redesign the section, I would like to go ahead and find out the exact reason why. I checked the YAML page of the Argo CD operator tab and I found out that YAML does not accept < > values all. But if I use that logic, I may need to remove the brackets from each property, which may open another can of worms.

Reply:
Re: modifying the `<` and `>` values, if you'd like we can do it as part of a separate PR, to unblock this PR. 

In general, it would be good to know why these are used. My follow up questions would be:
- Is this an OpenShift Docs convention? e.g. something that is done for all OpenShift documentation
    - If so, is there someone we can ask to figure out when `<` and `>` should be used?
- If not, was this something the GitOps team unilaterally invented? If so, we could probably look at who initially contributed these values, and ask.

In either case, we should not keep doing the wrong thing just because we did it in the past. We should figure what the right thing to do is, and start doing that.

Reply:
Yes, we can track this activity in a separate PR as I can move this PR further on the review ladder.